api/Navigator.json

```diff
@@ -2991,10 +2991,12 @@
               ]
             },
             "firefox": {
-              "version_added": "2"
+              "version_added": "2",
+              "notes": "Firefox 98 adds support for new safelisted schemes: <code>ftp</code>, <code>sftp</code>, <code>ftps</code>. All other safelisted schemes listed in the specification are supported, as is the custom scheme protocol format."
```

Comment:
FF tracks the spec - it supports all [safelisted schemes](https://developer.mozilla.org/en-US/docs/Web/API/Navigator/registerProtocolHandler#permitted_schemes) and the standard format of the custom schemes as defined in the spec. The schemes listed here were just added to the spec, and this reflects the addition to FF in FF98

I added a note because that follows the pattern of the other browsers - which use notes to indicate the small subset of the scheme that they support, if any. 

The other way to do this is to create subfeatures for all the schemes and for the title attribute, and indicate precisely which schemes are supported. I assume this was not done originally because there are 21 schemes. However that seems more accurate/reliable. 
@ddbeck Should I move to that approach or do you see it as "not worth the effort"

Reply:
You're right. That approach would be much more useful to readers: knowing which browsers support the one scheme you want to use seems like an obvious use case. Would you mind breaking them out?